module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -56,22 +57,33 @@ Read the [`miQC` package documentation](https://rdrr.io/github/greenelab/miQC/),
 
 
 | Library/Package      | Piece of Code      | What it's called    | What it does  |
-|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
+|----------------------|--------------------|---------------------|---------------|
 | `miQC`| [`mixtureModel()`](https://rdrr.io/github/greenelab/miQC/man/mixtureModel.html)| Mixture model | Fit a `miQC` mixture model to a `SingleCellExperiment` object for use in filtering |
 | `miQC`| [`filterCells()`](https://rdrr.io/github/greenelab/miQC/man/filterCells.html)| Filter cells | Filter cells from a `SingleCellExperiment` object based on a `miQC` model, returning a filtered `SingleCellExperiment` object |
 | `miQC`| [`plotMetrics()`](https://rdrr.io/github/greenelab/miQC/man/plotMetrics.html)| Plot metrics | Plot percent of mitochondrial reads against the number of unique genes found for each cell |
 | `miQC`| [`plotModel()`](https://rdrr.io/github/greenelab/miQC/man/plotModel.html)| Plot model | `miQC::plotMetics()` with the `miQC` fitted model overlaid |
 | `miQC`| [`plotFiltering()`](https://rdrr.io/github/greenelab/miQC/man/plotFiltering.html)| Plot filtering | Plot percent of mitochondrial reads against the number of unique genes found, coloring points based on whether they will be filtered out or not |
 
 
+## `SingleR`
+
+Read the [`SingleR` package documentation](https://rdrr.io/bioc/SingleR/), and a [e-book on its usage](http://bioconductor.org/books/release/SingleRBook/).
+
+| Library/Package      | Piece of Code      | What it's called    | What it does  |
+|----------------------|--------------------|---------------------|---------------|
+| `SingleR` | [`SingleR()`](https://rdrr.io/bioc/SingleR/man/SingleR.html) | Annotate scRNA-seq data | Combines the below "training" and classification functions into a single function to assign cell types to the cells in an SCE object using an annotated reference dataset |
+| `SingleR` | [`trainSingleR`](https://rdrr.io/bioc/SingleR/man/trainSingleR.html) | Train the SingleR classifier | Build a `SingleR` classifier model object from an annotated reference dataset |
```

Comment:
"training" vs (no quote) train here, I understand the dilemma but we should pick one flavor